pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -101,9 +107,20 @@ social:
                 = view :hoc_action_buttons, hoc_mode: hoc_mode
 
             #remainder.hidden-xs{style: "text-align: center; margin-top: 15px;"}
-              %div.watch-video-small{onclick: "return startVideo();"}
-                =hoc_s(:front_watch_regular_video)
-                %img{src: "/images/play-button.png", width: "25", onclick: "return startVideo();", style: "cursor: pointer;"}
+              -# If language is Espanol(Mexico), show alternate promo video
+              -# for LATM Hour of Code.
+              - unless @language == "la"
+                %div.watch-video-small{onclick: "return startVideo();"}
+                  =hoc_s(:front_watch_regular_video)
+                  %img{src: "/images/play-button.png", width: "25", onclick: "return startVideo();", style: "cursor: pointer;"}
+              - if @language == "la"
+                %div.watch-video-small{onclick: "return startLATAMVideo();"}
+                  =hoc_s(:front_watch_regular_video)
+                  %img{src: "/images/play-button.png", width: "25", onclick: "return startLATAMVideo();", style: "cursor: pointer;"}
+              - else
```

Comment:
using all three of `unless`, `if`, and `else` seems redundant, unless I'm misunderstanding something; won't this render the video twice for non-LA?

Reply:
Done!